### ISSUE TITLE
ansible-pull breaks includes with sudo:yes in 1.9.0.1

### ISSUE THREAD

**Issue by colthreepv** [ansible/ansible#10578]
Environment: ubuntu 14.04
ansible: 1.9.0.1, bug not present in 1.8.2 and 1.7.2

Steps to reproduce: use [those files](https://gist.github.com/mrgamer/efc77c0c00aba144d966)
then call:

```
$ ansible-pull -d testdir -U git@gist.github.com:/efc77c0c00aba144d966.git -i ../hosts.list -f ../main.yml
```

You can see that the first 2 tasks have explicit sudo and they are completed successfully, the other 2 fails.

If you execute the same `main.yml` with `ansible` instead of `ansible-pull`, it runs flawlessly

**Comment by sivel** [comment]
I believe this should be fixed by #10566

**Comment by colthreepv** [comment]
oh... awesome :-/ just have to find out how to revert back until then...

If anyone haves suggestions
